plugins/todo-backend/src/service/TodoReaderService.test.ts

```diff
@@ -361,11 +360,11 @@ describe('TodoReaderService', () => {
     });
     const service = new TodoReaderService({ todoReader, catalogClient });
 
-    await expect(service.listTodos({ entity: entityName })).rejects.toEqual(
-      expect.objectContaining({
-        name: 'InputError',
-        message: `Invalid entity source location type for component:default/my-component, got 'file'`,
-      }),
-    );
+    await expect(service.listTodos({ entity: entityName })).resolves.toEqual({
+      items: [],
+      totalCount: 0,
+      offset: 0,
+      limit: 10,
+    });
   });
```

Comment:
Let's keep these tests as they are, we don't support file locations or missing locations